sources/packages/web/src/views/aest/institution/Profile.vue

```diff
@@ -79,7 +78,7 @@
           </content-group></v-col
         >
       </v-row>
-      <p class="category-header-large color-blue mt-2 mb-2">Mailing address</p>
```

Comment:
When there is no data returned e.g address line 2, how it is suppose to be shown? just leave it to show blank or use - like figma. @hellolynn-tbtb ?

![image](https://user-images.githubusercontent.com/54600590/195948585-60032ed0-f8e6-435d-a932-8ec65b32735c.png)

Reply:
Created a composable to address this empty string issue.

Reply:
Thanks for adding the composable.